jduong09/thirteen
# Issue 8

# Feature

## Card Function

### Summary
- Created function that takes in a suit and rank, creates 'card' object
- Created javascript file that simulates creation of deck that contains 52 'cards'

### How To Test
- Pull down, install dependencies.
   - `npm install`
- Execute file ** dummyDeck.js **
   - `node components/utilities/dummyDeck.js`
- Validate terminal log
   - Should... 
      - [x] Display array of 52 cards 
      - [x] Number of cards: 52
      - [x] First card is Ace of Hearts
      - [x] 19th card is Seven of Diamonds


## Hand Component

### Summary
- Created component that takes in 3 props.
   - Array of random cards, which will be hands cards for game cycle
   - Function to send hands turn back to game component
   - Function to pass turn
- Displays list of cards on browser.
   - Highlights cards that will be chosen for hand's turn.  

### How To Test
- Open localhost:3000
- Shuffle cards.
- Select cards to be used in combo.
- Finalize Turn and check log for array sent to game component.
